src/components/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (todo: string) => void,
+  updateTodo: (uTodo: Todo) => void,
+};
+
+export const Header: React.FC<Props> = ({ todos, addTodo, updateTodo }) => {
+  const isSomeActive = todos.some(todo => !todo.completed);
+
+  const [value, setValue] = useState('');
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    addTodo(value);
+    setValue('');
+  };
+
+  const handleCompleteAll = (e: React.MouseEvent) => {
+    e.preventDefault();
+    const activeTodos = todos.filter(todo => !todo.completed);
+
+    if (isSomeActive) {
```

Comment:
The part 'isSomeActive' is calculated at the beginning of the function and not updated later, so its value may not reflect the current state of todos when the 'handleCompleteAll' function is called.